src/actions/index.js

```diff
@@ -376,6 +378,14 @@ export function removeCommunityModerator (community, moderatorId, prevProps) {
   }
 }
 
+export function removeCommunityMember (community, userId, cacheId, prevProps) {
+  return {
+    type: REMOVE_COMMUNITY_MEMBER,
+    payload: {api: true, path: `/noo/community/${community.id}/member/${userId}`, method: 'DELETE'},
+    meta: {slug: community.slug, userId, cacheId, prevProps}
```

Comment:
ok, I'll give it a shot, with a time limit.

On Tue, Jul 12, 2016 at 1:16 PM, Lawrence Wang <notifications@github.com>
wrote:

> In src/actions/index.js
> <https://github.com/Hylozoic/hylo-redux/pull/76#discussion_r70513553>:
>
> > @@ -376,6 +378,14 @@ export function removeCommunityModerator (community, moderatorId, prevProps) {
> >    }
> >  }
> >
> > +export function removeCommunityMember (community, userId, cacheId, prevProps) {
> > +  return {
> > +    type: REMOVE_COMMUNITY_MEMBER,
> > +    payload: {api: true, path: `/noo/community/${community.id}/member/${userId}`, method: 'DELETE'},
> > +    meta: {slug: community.slug, userId, cacheId, prevProps}
>
> yeah -- unless you want to take a stab at the middleware
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/76/files/17410bee3d125958c8ae4e94b3dabcef767825a6#r70513553>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA2Y9CyCUaK94WRO41_VQeDW8txZoz5oks5qU_YjgaJpZM4JIaXw>
> .
>



-- 
Robbie Carlton

719.358.3482
<robbie@authenticmanprogram.com>